src/site/content/en/blog/payment-and-address-form-best-practices/index.md

```diff
@@ -0,0 +1,782 @@
+Bear this in mind when designing address forms—you should consider alternatives such as using a 
+single `textarea` (see below) or other options.
+{% endAside %}
+
+
+#### Consider using a single textarea for address {: #address-textarea}
+
+The most flexible option for addresses is to provide a single `textarea`.
+
+The `textarea` approach fits any address format, and it's great for cutting and pasting—but bear in 
+mind that it may not fit your data requirements, and users may miss out on autofill if they previously 
+only used forms with `address-line1` and `address-line2`.
+
+For a textarea, use `street-address` as the autocomplete value.
+
+Here is an example of a form that demonstrates the use of a single `textarea` for address:
+
+{% Glitch {
+  id: 'address-form',
+  path: 'index.html',
+  height: 480
+} %}
+
+#### Internationalize and localize your address forms {: #internationalization-localization} 
+
+It's especially important for address forms to consider [internationalization and localization](https://www.smashingmagazine.com/2020/11/internationalization-localization-static-sites/), depending 
+on where your users are located. 
+
+Be aware that the naming of address parts varies as well as address formats, even within the same 
+language.
+
+```text
+    ZIP code: US
+ Postal code: Canada
+    Postcode: UK
+     Eircode: Ireland
+         PIN: India
+```
+
+It can be irritating or puzzling to be presented with a form that doesn't fit your address or that 
+doesn't use the words you expect. 
+
+Customizing address forms [for multiple locales](https://www.smashingmagazine.com/2020/11/internationalization-localization-static-sites#determining-user-s-language-and-region) may be necessary for your site, but using techniques 
+to maximize form flexibility (as described above) may be adequate. If you don't localize your address 
+forms, make sure to understand the key priorities to cope with a range of address formats:
+* Avoid being over-specific about address parts, such as insisting on a street name or house number.
+* Where possible avoid making fields `required`. For example, addresses in many countries don't 
+have a postal code, and rural addresses may not have a street or road name.
+* Use inclusive naming: 'Country/region' not 'Country'; 'ZIP/postal code' not 'ZIP'.
+
+Keep it flexible!
+
+{% Aside %}
+The [simple address form example above](#address-textarea) can be adapted to work 'well enough' 
+for many locales.
+{% endAside %}
+
+#### Consider avoiding postal code address lookup {: #postal-code-address-lookup}
+
+Some websites use a service to look up addresses based on postal code or ZIP. This may be sensible 
+for some use cases, but you should be aware of the potential downsides.
+
+Postal code address suggestion doesn't work for all countries—and in some regions, post codes can 
+include a large number of potential addresses. 
+
+<figure class="w-figure">
```

Comment:
This GIF hinted me to use `datalist` instead of `select` element. It's relatively widely available https://caniuse.com/datalist.

Reply:
Ah yes — I'm a big fan of datalist. Are you suggesting we use that for country, or something else?

Comparison (for years): https://datalist-select.glitch.me

I also made this as a prototype for what a standardized HTML element for multiple selections might look like: https://glitch.com/~multi-input

Reply:
The website can use `datalist` to autocomplete from the long list. Autofill does work too though it requires user's prior entry of the same address.